redhat-developer/developers.redhat.com
# [DEVELOPER-5496] Sign-up hero assembly

### JIRA Issue Link
[DEVELOPER-5496 Sign Up hero assembly](https://issues.jboss.org/browse/DEVELOPER-5496)

To test this assembly:

1. Go to create a simple page
2. Add the sign up hero assembly
3. Fill out the required fields
4. Once the page is saved, the assembly should be vissible to unauthenticated users only
